Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -305,23 +509,28 @@ def route_list_values() -> Response:
                 demisto.debug(err_msg)
                 return Response(err_msg, status=401)
 
-        limit, offset, out_format, query = get_request_args(params)
+        request_args = get_request_args(params)
 
         values = get_outbound_ioc_values(
-            out_format=out_format,
+            out_format=request_args.get('out_format'),
             on_demand=params.get('on_demand'),
-            limit=limit,
-            offset=offset,
+            limit=request_args.get('limit'),
+            offset=request_args.get('offset'),
             last_update_data=demisto.getIntegrationContext(),
-            indicator_query=query,
-            cache_refresh_rate=params.get('cache_refresh_rate')
+            indicator_query=request_args.get('query'),
+            cache_refresh_rate=params.get('cache_refresh_rate'),
+            mwg_type=request_args.get('mwg_type'),
+            strip_port=request_args.get('strip_port'),
+            drop_invalids=request_args.get('drop_invalids'),
+            category_default=request_args.get('category_default'),
+            category_attribute=request_args.get('category_attribute')
         )
 
         mimetype = get_outbound_mimetype()
         return Response(values, status=200, mimetype=mimetype)
 
     except Exception as e:
-        return Response(str(e), status=400, mimetype='text/plain')
+        return Response(str(e) + traceback.format_exc(), status=400, mimetype='text/plain')
```

Comment:
Please add a new line after str(e)

Reply:
```suggestion
        return Response(traceback.format_exc(), status=400, mimetype='text/plain')
```
The traceback should include the error